### PR TITLE
Ngslinker

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/irida-linker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - perl-file-path
+    - perl-config-simple
+    - perl-termreadkey
+    - perl-http-message
+    - perl-libwww-perl
+test:
+  commands:
+    - ngsArchiveLinker.pl
+extra:
+  skip-lints:
```
Comment (bgruening) [inline recipes/irida-linker/meta.yaml]: You can skip these skips, you follow our best practices very much! :)

File: recipes/irida-linker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  url: https://github.com/phac-nml/irida-linker/archive/1.0.1.tar.gz
+  md5: 958689540fb5fde2746af9354438d695
+  
+build:
+  number: 0
+  skip: True  # [osx]
+  
+requirements:
+  build:
+    - perl-lwp-simple
```
Comment (bgruening) [inline recipes/irida-linker/meta.yaml]: You should add `perl` as dependency for build and run ...

File: recipes/irida-linker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  version: "1.0.1"
+
+source:
+  fn: 1.0.1.tar.gz
+  url: https://github.com/phac-nml/irida-linker/archive/1.0.1.tar.gz
+  md5: 958689540fb5fde2746af9354438d695
+  
+build:
+  number: 0
+  skip: True  # [osx]
```
Comment (bgruening) [inline recipes/irida-linker/meta.yaml]: Is skipping osx really needed?

File: recipes/irida-linker/build.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+cp ngsArchiveLinker.pl $PREFIX/bin
+
+#testing usage statement
+perl ngsArchiveLinker.pl
```
Comment (cbrueffer) [inline recipes/irida-linker/build.sh]: Can this go away, given that ngsArchiveLinker.pl is already tested in meta.yaml?

File: recipes/irida-linker/meta.yaml (hunk 1 above)
Comment (Takadonet) [reply recipes/irida-linker/meta.yaml]: Removed.

File: recipes/irida-linker/meta.yaml (hunk 3 above)
Comment (Takadonet) [reply recipes/irida-linker/meta.yaml]: Will now build osx.

File: recipes/irida-linker/build.sh (hunk 4 above)
Comment (Takadonet) [reply recipes/irida-linker/build.sh]: It went away! Is already tested in meta.yaml

**Pull request merged**